### ISSUE TITLE
Monitor: make accrual detection useful (or remove)

### ISSUE THREAD

**Issue by hsanjuan** [ipfs/ipfs-cluster#939]
**Describe the feature you are proposing**

Accrual failure detection exists in the `monitor` module. However, two metrics coming unusually close (like in a re-start), would skew the interval between metrics and then trigger alerts because the next one would come at a normal interval.

As a quick mitigation, safeguards were added to never alert if the last metric was still valid per it's TTL. But since metrics are valid 2x the interval in which they are produced, this means that accrual detection only kicks in on a full degradation (a metric is not only coming in a bit late, but very late), and just alerts right away. The edge cases in which it does not alert right away are very narrow (and honestly, at that point there should be an alert!).

In practice, metrics validity should not depend on their hardcoded TTL but rely on accrual detection which effectively handles some degradation. If a peer is re-started it could send metrics with a new epoch or in a way that accrual can detach from previous series.

In short, accrual is a complicated thing and we have it for no value right now, we should take full advantage of it or drop it as useless complexity if it turns out it can't really help: it should reduce the number of false positives when making alert decisions purely based on TTL (and not add more false positives).

**Comment by kishansagathiya** [comment]
We could also ignore metrics received at anomalous intervals from the distribution

**Comment by hsanjuan** [comment]
> We could also ignore metrics received at anomalous intervals from the distribution

No, metrics are sent for a reason. For example, to inform of a new freespace value when the storage is decreasing rapidly.

**Comment by kishansagathiya** [comment]
I think that came out wrong. Let me make that clear.
Don't ignore the metric, but do not count it in the distribution if it came at an anomalous interval. (sort of like how data scientists clear the data before running models on them)
